README.md

```diff
@@ -46,21 +42,23 @@ or with a CDN:
-| [`trackInAppDelivery`](#trackInAppDelivery)  	| Track when a message has been delivered to a user's device                                                                	|
-| [`trackInAppOpen`](#trackInAppOpen)      	| Track when a message is opened and marks it as read                                                                       	|
-| [`trackPurchase`](#trackPurchase)       	| Track purchase events                                                                                                     	|
-| [`updateCart`](#updateCart)          	| Update _shoppingCartItems_ field on user profile                                                                          	|
-| [`updateSubscriptions`](#updateSubscriptions) 	| Updates user's subscriptions                                                                                              	|
-| [`updateUser`](#updateUser)          	| Change data on a user's profile or create a user if none exists                                                           	|
-| [`updateUserEmail`](#updateUserEmail)     	| Change a user's email and reauthenticate user with the new email address (in other words, we will call `setEmail` for you)                                                                                             	|
+| Method Name                                            | Description                                                                                                                         |
+| ------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------- |
+| [`getInAppMessages`](#getInAppMessages)                | Either return in-app messages as a Promise or automatically paint them to the DOM if the second argument is passed `DisplayOptions` |
+| [`initialize`](#initialize)                            | Method for identifying users and setting a JWT                                                                                      |
+| [`track`](#track)                                      | Track custom events                                                                                                                 |
+| [`trackInAppClick`](#trackInAppClick) :rotating_light: | Track when a user clicks on a button or link within a message                                                                       |
+| [`trackInAppClose`](#trackInAppClose)                  | Track when an in-app message is closed                                                                                              |
+| [`trackInAppConsume`](#trackInAppConsume)              | Track when a message has been consumed. Deletes the in-app message from the server so it won't be returned anymore                  |
```

Comment:
@pauljung14 Can we give any guidance as to when this should be called? Right after displaying the message?